issue_03/script.js

```diff
@@ -0,0 +1,16 @@
+const ul = document.getElementById('js-ul');
+const hrefArray = ['a1.html', 'a2.html'];
+const textArray = ['a1', 'a2'];
+const srcArray = ['/img/bookmark.png', '/img/message.png'];
+
+for (let i = 0; i < 2; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+
+    a.setAttribute('href', hrefArray[i]);
+    a.textContent = textArray[i];
+    img.setAttribute('src', srcArray[i]);
```

Comment:
```suggestion
    a.href = array[i].href;
    a.textContent = array[i].text;
    img.src = array[i].src;
```
nits
You can also write it this way.

Reply:
Thank you for pointing that out.
It's fixed :sparkles:
https://github.com/yoshikawa-akr/js-lessons/pull/3/commits/21fe2815c83b383b40c3390ec092bcf16b13592c